### PR TITLE
Added .travis.yml

### LINKED ISSUE #20: Set up travise
Currently travis is not set up and test are not executed on PR and commits

### PR DESCRIPTION
**Pull request by notgiorgi**
Closes #20

I also added travis and npm badges

### CODE REVIEW

**Comment by safareli** [comment]
I have enabled travis on this repo can I think if you push something it will trigger build

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+language: node_js
+notifications:
+  email: false
+node_js:
+  - '4'
+install: npm install
+script:
+  - npm run check
+  - npm run bench
+branches:
```
Comment (safareli) [inline .travis.yml]: Why this?

Comment (notgiorgi) [inline .travis.yml]: Which one, `bench`?

**Comment by notgiorgi** [comment]
Build failed, we need higher version of node I guess.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  only:
+    - master
```
Comment (davidchambers) [inline .travis.yml]: Nit: Please add a trailing newline.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+[![npm](https://img.shields.io/npm/v/daggy.svg)](https://www.npmjs.com/package/daggy)
+
 Library for creating tagged constructors a.k.a. "union types" or "sum types".
 
 ## `daggy.tagged(typeName, fields)`
 
 Creates a new constructor with the given field names
 
-```javascript
+```js
```
Comment (davidchambers) [inline README.md]: Please revert this change.

File: .travis.yml (hunk 1 above)
Comment (safareli) [inline .travis.yml]: I think we can just remove it :d

File: README.md [hunk 4]
```diff
@@ -1,12 +1,15 @@
 # Daggy
 
+[![Build Status](https://travis-ci.org/fantasyland/daggy.svg?branch=master)](https://travis-ci.org/fantasyland/daggy)
+[![npm](https://img.shields.io/npm/v/daggy.svg)](https://www.npmjs.com/package/daggy)
```
Comment (davidchambers) [inline README.md]: Let's use Shield.io for both badges to ensure visual consistency.

File: README.md (hunk 3 above)
Comment (notgiorgi) [inline README.md]: ofc

File: .travis.yml (hunk 2 above)
Comment (davidchambers) [inline .travis.yml]: Thanks. :)

File: README.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Daggy
 
+[![Build Status](https://img.shields.io/travis/fantasyland/daggy.svg?branch=master)](https://travis-ci.org/fantasyland/daggy)
```
Comment (davidchambers) [inline README.md]: I believe the correct format is `https://img.shields.io/travis/USER/REPO/BRANCH.svg`.

File: .travis.yml (hunk 1 above)
Comment (davidchambers) [inline .travis.yml]: Why only test `master`?

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: node_js
+notifications:
+  email: false
+node_js:
+  - '6'
+install: npm install
```
Comment (davidchambers) [inline .travis.yml]: I think this is the default command, so we could omit it.

**Comment by davidchambers** [comment]
This is looking good, @notgiorgi. Could you squash the commits?

File: .travis.yml (hunk 6 above)
Comment (notgiorgi) [inline .travis.yml]: I think this is more explicit

**Comment by notgiorgi** [comment]
Done

File: .travis.yml (hunk 6 above)
Comment (notgiorgi) [inline .travis.yml]: They're same, you can just write single command on the same line, but multiple should be a list.

initially `script` consisted of 2 commands, that's why, I can make both single line.
